docs/community/newDiagram.md

```diff
@@ -17,20 +21,17 @@ For instance:
 
 There are some jison specific sub steps here where the parser stores the data encountered when parsing the diagram, this data is later used by the renderer. You can during the parsing call a object provided to the parser by the user of the parser. This object can be called during parsing for storing data.
 
-```jison
-statement
-	: 'participant' actor  { $$='actor'; }
-	| signal               { $$='signal'; }
-	| note_statement       { $$='note';  }
-	| 'title' message      { yy.setTitle($2);  }
-	;
-```
+    statement
+    	: 'participant' actor  { $$='actor'; }
+    	| signal               { $$='signal'; }
+    	| note_statement       { $$='note';  }
+    	| 'title' message      { yy.setTitle($2);  }
+    	;
```

Comment:
jison codeblock removed?

Reply:
We dont have a highlight for now. and it brakes the doc build.

Reply:
There should be some option to ignore jison. 
Even if it's not possible, we should keep jison in source docs and strip just the jison language tag in transformed docs, keeping the code fences.